### PR TITLE
Make artifact of linting output

### PR DESCRIPTION
**Pull request by Jan200101**
related to #41 
I don't know what part of the circleci script is for testing
I'll happily add a artifact for that if someone can point it out for me

### CODE REVIEW

**Comment by bgauduch** [comment]
Hello @Jan200101 ! Thanks a lot for your contribution 👍

There is no test stage in the CI yet, it is planned as per issue #15 

The artifact setup looks fine, but the way the linting script is called seem's to be an issue, as it produce empty results.
It can be seen in the circleci job [here](https://circleci.com/gh/Zenika/terraform-azure-cli/109), in the `uploading artifacts` stage:
```
Uploading /tmp/linter to tmp/linter
Uploading /tmp/linter/lint-error: SKIPPED, file is empty
Uploading /tmp/linter/lint-output: SKIPPED, file is empty
```

I will take a look at this when a get time, but do not hesitate to dig into this as well 😉

**Comment by bgauduch** [comment]
fixes #41

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,7 +7,12 @@ jobs:
       - checkout
       - run:
           name: Lint dockerfile
-          command: ./scripts/lint.sh
+          command: |
+            mkdir /tmp/linter
+            ./scripts/lint.sh 2> /tmp/linter/lint-error > /tmp/linter/lint-output
```
Comment (bgauduch) [inline .circleci/config.yml]: The way the script is called produce no output, need some digging

**Comment by Jan200101** [comment]
I assumed this is may only be a matter of docker not outputting to stdout and stderr but there seems to be more behind it

**Comment by Jan200101** [comment]
Found the actual cause of why nothing was outputting
the Dockerfile was just too good and the linter had nothing to complain about

I setup a branch and added a random character into the Dockerfile to break it on purpose.
https://circleci.com/gh/Jan200101/terraform-azure-cli/9
it outputs just fine but I had to install ca-certificates for it to be able to upload the artifact

**Comment by Jan200101** [comment]
right now the linter outputs nothing to STDOUT or STDERR and only into the file
should it output to both?

**Comment by pyaillet** [comment]
> right now the linter outputs nothing to STDOUT or STDERR and only into the file
> should it output to both?

Well thought ! Indeed outputing to both the file and STD{OUT,ERR} would be neat.

**Comment by Jan200101** [comment]
latest commit makes it output to the commandline and a file
I also merged both stderr and stdout into a single file

https://app.circleci.com/jobs/github/Jan200101/terraform-azure-cli/14

**Pull request merged**